src/list/src/listScript.sml

```diff
@@ -1136,10 +1136,10 @@ Proof
 GEN_TAC THEN Induct THEN SRW_TAC[] [FOLDR, MAP]
 QED
 
-Theorem LENGTH_TL:
-   !l. 0 < LENGTH l ==> (LENGTH (TL l) = LENGTH l - 1)
+Theorem LENGTH_TL[simp]:
+   !l. (LENGTH (TL l) = LENGTH l - 1)
```

Comment:
I think it should be possible to remove the parentheses now that the implication is gone.